test/Microsoft.Sbom.Parsers.Spdx22SbomParser.Tests/Parser/SbomRelationshipParserTests.cs

```diff
@@ -40,7 +40,6 @@ public void StreamEmptyTestReturnsNull()
         var result = this.Parse(parser);
     }
 
-    [DataTestMethod]
```

Comment:
For consistency, can you please put the `[TestMethod]` attribute _before_ the `DataRow` attributes for the tests in this file?